jans-cedarling/cedarling/src/entity_builder/build_expr.rs

```diff
@@ -42,7 +42,27 @@ impl TknClaimAttrSrc {
         &self,
         src: &Value,
     ) -> Result<Option<RestrictedExpression>, BuildExprErrorVec> {
-        build_expr_from_value(&self.expected_type, src)
+        // Special handling for aud claims: convert string to array if needed
+        if self.claim == "aud" && matches!(self.expected_type, ExpectedClaimType::Array(_)) {
```

Comment:
Maybe you also wanted to add check if claim type is String?

```suggestion
        if self.claim == "aud"
            && matches!(
                self.expected_type,
                ExpectedClaimType::Array(_) | ExpectedClaimType::String
            )
        {
```